extension/src/background/helpers/migration.ts

```diff
@@ -74,13 +77,15 @@ export const migrateTrustlines = async ({
         }),
       );
 
-      sendTrustlineBalanceTx.addOperation(
-        Operation.payment({
-          destination: newKeyPair.publicKey,
-          asset,
-          amount: bal.balance,
-        }),
-      );
+      if (new BigNumber(bal.balance).gt("0")) {
+        sendTrustlineBalanceTx.addOperation(
```

Comment:
TIL: trying to send a payment of amount `0` actually causes `Operation.payment` to error